src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  allBooks: [],
+};
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.allBooks.push(action.payload);
+    },
+    removeBook: (state, action) => state.allBooks.filter((book) => book.id !== action.payload),
```

Comment:
- [ ] Nice job structuring the files. :+1: Since the array method `filter` does not mutate the original array it will not have any effect on the state. Kindly, assign the filtered result back to the state. ✍️

**Example**
```
state.allBooks = state.allBooks.filter((book) => book.id !== action.payload)
```